src/locales/ar-SA.json

```diff
@@ -362,29 +357,16 @@
       "required": "مطلوب",
       "website": "أدخل عنوان URL صالح"
     },
-    "noteDescription": {
-      "pre": "",
-      "link": "مستودع المكتبة العامة",
-      "post": "ليستخدمها الآخرون في رسوماتهم."
-    },
-    "noteGuidelines": {
-      "pre": "يجب الموافقة على المكتبة يدويًا أولاً. يرجى قراءة ",
-      "link": "الإرشادات",
-      "post": ""
-    },
-    "noteLicense": {
-      "pre": "",
-      "link": "رخصة إم أي تي ",
-      "post": "وهو ما يعني باختصار أنه يمكن لأي شخص استخدامها دون قيود."
-    },
```

Comment:
I'm thinking we should not migrate RTL languages as it's likely significant that they didn't translate pre/post strings. Running it through a translator, it looks weird though, so I think it's best if people re-translate the whole string.

So let's revert the migrations for all RTL languages: ar-SA, fa-IR, he-IL.

Reply:
Got it, reverted in 10e2284